src/app/assets/styles/_color_variables.scss

```diff
@@ -16,3 +16,5 @@
 $primary: #326DE6;
 $line: #AAAAAA;
 $subline: #888888;
+$link: #326DE6;
+$hoverLink: #1254DF;
```

Comment:
Actually, I think the primary color will work O.K., no need for a variation here.